examples/dk300_edge_system_iotcc.py

```diff
@@ -172,27 +188,36 @@ def simulated_device():
     #        another Resource in IoTCC of which the should be the child of a parent-child relationship among Resources
     ram_device = SimulatedDevice(config['DeviceName'], "Device-RAM")
     reg_ram_device = iotcc.register(ram_device)
-    iotcc.create_relationship(reg_edgesystem, reg_ram_device)
+    if reg_ram_device is None:
+        print "Device registration to IOTCC failed"
+        exit()
```

Comment:
I don't get this. Why if we could not register a device do we have the program exit?